### PR TITLE
add se_dim keyword

### LINKED ISSUE #52: add control over single epoch dims
with keywords, currently it is calculated from coadd dims

### PR DESCRIPTION
**Pull request by esheldon**
closes #52 

se_dim will be set, only check is that the image size isn't
smaller than 2*edge width

added test

### CODE REVIEW

**Comment by esheldon** [comment]
ok with you @beckermr ?

File: descwl_shear_sims/simple_sim.py [hunk 1]
```diff
@@ -262,13 +266,18 @@ def __init__(
...
-        # the SE image could be rotated, so we make it big enough to cover the
-        # whole coadd region plus we make sure it is odd
-        self.se_dim = (
-            int(np.ceil(self.coadd_dim * np.sqrt(2))) + 10 + 2*edge_width
-        )
-        if self.se_dim % 2 == 0:
-            self.se_dim = self.se_dim + 1
+        if se_dim is not None:
+            self.se_dim = int(se_dim)
+            assert self.se_dim > 2*edge_width
```
Comment (beckermr) [inline descwl_shear_sims/simple_sim.py]: Shouldn't this lower limit be bigger?

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py]: I guess we could add an arbitrary number to it

Comment (beckermr) [reply descwl_shear_sims/simple_sim.py]: Well we have the coadd dim and we might have rotations, so I think the default is the min.

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py]: But the point of the se_dim keyword is so the user can set it to whatever size they want, even something that could have an edge in it and get rejected later

Comment (beckermr) [reply descwl_shear_sims/simple_sim.py]: But this keyword is set for all epochs. Why would we want to reject all of them?

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py]: good point

**Pull request merged**